features/kiesrecht.feature

```diff
@@ -7,116 +7,79 @@ Functionaliteit: Kiesrecht
-	Gegeven op de PL van een persoon is categorie 13, element 38.10 gelijk aan A (= persoon is uitgesloten)
-	En categorie 13, element 38.20 is een datum in het verleden
-	Als de persoon wordt geraadpleegd
-	Dan is uitgeslotenVanKiesrecht niet aanwezig
-	En is einddatumUitsluitingKiesrecht niet aanwezig
-
-Scenario: de persoon is uitgesloten van kiesrecht en de einddatum uitsluiting kiesrecht ligt in de toekomst
-	Gegeven op de PL van een persoon is categorie 13, element 38.10 gelijk aan A (= persoon is uitgesloten)
-	En categorie 13, element 38.20 is een datum in de toekomst
-	Als de persoon wordt geraadpleegd
-	Dan is uitgeslotenVanKiesrecht gelijk aan true
-	En is einddatumUitsluitingKiesrecht gelijk aan de waarde van element 38.20
-
-Abstract Scenario: de einddatum uitsluiting kiesrecht is geheel of deels onbekend
-	Gegeven op de PL van een persoon is categorie 13, element 38.10 gelijk aan A (= persoon is uitgesloten)
-	En op de PL van een persoon is categorie 13, element 38.20 gelijk aan <geheel of deels onbekende datum>
-	Als de persoon wordt geraadpleegd op 6 september 2019
-	Dan is uitgeslotenVanKiesrecht <europeesKiesrecht>
-	En is einddatumUitsluitingEuropeesKiesrecht <einddatumUitsluitingEuropeesKiesrecht>
-
-	Voorbeelden:
-	| geheel of deels onbekende datum | uitgeslotenVanKiesrecht | einddatumUitsluitingEuropeesKiesrecht |
-	| 00000000                        | true                    | niet aanwezig                         |
-	| 20190000                        | true                    | jaar: 2019                            |
-	| 20180000                        | niet aanwezig           | niet aanwezig                         |
-	| 20191000                        | true                    | jaar: 2019, maand: 10                 |
-	| 20190900                        | true                    | jaar: 2019, maand: 09                 |
-	| 20190800                        | niet aanwezig           | niet aanwezig                         |
+Rule: Uitsluiting van Europees kiesrecht wordt alleen opgenomen wanneer de einddatum uitsluiting in de toekomst ligt
+	- Wanneer alleen het jaar van de einddatum uitsluiting bekend is, dan wordt de uitsluiting opgenomen tot en met dat jaar.
+	- Wanneer het jaar en de maand van de einddatum uitsluiting bekend is, en de dag niet, dan wordt de uitsluiting opgenomen tot en met die maand.
+	- Een volledig onbekende datum wordt hetzelfde geïnterpreteerd en weergegeven als het niet aanwezig zijn van die datum.
+
+	Abstract Scenario: Europees kiesrecht <omschrijving>
+		Gegeven het systeem heeft een persoon met de volgende gegevens
+		| naam                        | waarde     |
+		| burgerservicenummer         | 999990001  |
+		En de persoon heeft de volgende kiesrecht gegevens
+		| naam                                          | waarde                                  |
+		| europeesKiesrecht (31.10)                     | <Europees kiesrecht>                |
```

Comment:
Mag de GBA API dit als boolean teruggeven ipv 1 of 2?

Reply:
ik denk het wel. Het betekent precies hetzelfde.